docs/03-architecture/supported-browsers.mdx

```diff
@@ -43,7 +43,63 @@ In addition, to reduce bundle size, Next.js will only load these polyfills for b
 
 If your own code or any external npm dependencies require features not supported by your target browsers (such as IE 11), you need to add polyfills yourself.
 
+<PagesOnly>
 In this case, you should add a top-level import for the **specific polyfill** you need in your [Custom `<App>`](/docs/pages/building-your-application/routing/custom-app) or the individual component.
+</PagesOnly>
+
+<AppOnly>
+ 
+To include polyfills, you can import them into the [`instrumentation-client.js` file](/docs/app/api-reference/file-conventions/instrumentation-client).
+
+For other cases, you can create a client module that loads the polyfills you need. For example if you only want to polyfill client side:
```

Comment:
This paragraph doesn't seem right here since it comes right before the instrumentation-client code snippet